src/components/App.js

```diff
@@ -206,7 +268,8 @@ export default connect(
   state => ({
     selectedSource: getSelectedSource(state),
     startPanelCollapsed: getPaneCollapse(state, "start"),
-    endPanelCollapsed: getPaneCollapse(state, "end")
+    endPanelCollapsed: getPaneCollapse(state, "end"),
+    activeSearchState: getActiveSearch(state)
```

Comment:
nit: it feels a bit off to have renamed the selector to `getActiveSearch` and use a `activeSearchState` prop. Renaming it to `activeSearch` would be more cnosistent IMO